Sprint-1/interpret/to-pounds.js

```diff
@@ -4,24 +4,53 @@ const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
 );
+// Removing the "p" in the end
 
 const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+console.log(paddedPenceNumberString);
+// This code makes sure the number has at least 3 digits by adding zeros at the start
+
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
 );
+// This code takes everything except the last two digits
 
 const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
   .padEnd(2, "0");
+//  This code takes the last two numbers for pence and makes sure it has 2 digits
```

Comment:
Can we expect the code to work as intended even if we deleted `.padEnd(2, "0")` from the code?

Reply:
Hello @cjyuan,
Yes, you are right. The code will still work as intended without .padEnd(2, "0"). That is extra logic.